scripts/output_code_size.sh

```diff
@@ -0,0 +1,80 @@
+#!/bin/bash
+
+SCRIPT_DIRECTORY="$(dirname "$(realpath "$0")")"
+. ${SCRIPT_DIRECTORY}/helper_find_llvm_install.sh
+
+OBJDUMP=$(find_llvm_tool llvm-objdump)
+
+if [ ! -x ${OBJDUMP} ] ; then
```

Comment:
It would be nice to have a second argument to `find_llvm_tool` for whether to fail if the tool is not found.  I think in all of our current uses we want to print an error and have the script exit with a failure exit code if it can't find the tool.

Reply:
Agree, I will do that as a wrapper.